getting-started/index.md

```diff
@@ -74,6 +74,10 @@ just remember to import them in `main.less` using the correct path.
 If you don't want to use the generator, you can just download the source files and
 import them into your project.
 
+> NOTE: Be sure to run the less files through
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/355/commits/62add6da5651839d9c23ec441ba77f4e36976804